save-frontend-common/src/main/kotlin/com/saveourtool/save/frontend/common/components/topbar/TopBar.kt

```diff
@@ -0,0 +1,80 @@
+/**
+ * Top bar of web page
+ */
+
+@file:Suppress("FILE_NAME_MATCH_CLASS", "FILE_WILDCARD_IMPORTS", "WildcardImport")
+
+package com.saveourtool.save.frontend.common.components.topbar
+
+import com.saveourtool.save.frontend.common.components.basic.languageSelector
+import com.saveourtool.save.frontend.common.externals.fontawesome.*
+import com.saveourtool.save.frontend.common.externals.fontawesome.FontAwesomeIconModule
+import com.saveourtool.save.frontend.common.externals.fontawesome.fontAwesomeIcon
+import com.saveourtool.save.frontend.common.utils.UserInfoAwarePropsWithChildren
+import com.saveourtool.save.frontend.common.utils.notIn
+import com.saveourtool.save.validation.FrontendRoutes
+
+import js.core.jso
+import react.*
+import react.dom.html.ButtonHTMLAttributes
+import react.dom.html.ReactHTML.button
+import react.dom.html.ReactHTML.nav
+import react.router.useLocation
+import web.cssom.ClassName
+import web.cssom.vw
+import web.html.ButtonType
+import web.html.HTMLButtonElement
+
+/**
+ * A component for web page top bar.
+ */
+val topBarComponent: FC<UserInfoAwarePropsWithChildren> = FC { props ->
```

Comment:
What I am thinking now is that we should have different TopBars, I guess.

Reply:
Splited it